Commands/HelpCommand.php

```diff
@@ -39,17 +39,20 @@ class HelpCommand extends UserCommand
     /**
      * @var string
      */
-    protected $version = '1.2.0';
+    protected $version = '1.2.1';
```

Comment:
Go for **1.3.0** here, as this isn't just a bug fix, but a functionality change.